root/static/styles/layout.less

```diff
@@ -126,8 +125,168 @@ pre code {
 /*
 ** Header
 */
-#header {
-    width: 100%;
+.header {
+    @menu-item-text-padding: 8px;
+    @logo-width: 166px;
+    @logo-height: 26px;
+
+    height: 62px;
+    margin-top: 10px;
+    margin-left: 16px;
+    margin-right: 16px;
+    clear: both;
+
+    img.logo {
+        width: @logo-width;
+        height: @logo-height;
+        margin-top: 16px;
+    }
+
+    .right {
+        width: 85%; // fallback for browsers without support for calc()
+        width: ~"calc(100% - @{logo-width} - 26px)";
+        float: right;
+
+        .top {
+            height: 31px;
+            border-bottom: @musicbrainz-purple 2px solid;
+            clear: both;
+            .links-container {
+                float: left;
+                ul.menu {
+                    li {
+                        img.gravatar {
+                            vertical-align: text-top;
+                        }
+                    }
+                }
+            }
+            .search-container {
+                @item-height: 25px;
+                @outer-border-color: fade(@musicbrainz-purple, 50%);
+                @inner-border-color: @medium-border;
+
+                float: right;
+                form {
+                    width: 302px;
+                    margin-top: 3px;
+                    input#headerid-query {
+                        width: 150px;
+                        height: @item-height;
+                        padding-left: 6px;
+                        padding-right: 6px;
+                        font-size: 1rem;
+                        border: 1px solid @outer-border-color;
+                        border-right-color: @inner-border-color;
+                    }
+                    select#headerid-type {
+                        width: 116px;
+                        height: @item-height;
+                        font-size: 1rem;
+                        background-color: #fff;
+                        border: 1px solid @outer-border-color;
+                        border-left-color: @inner-border-color;
+                        border-right-color: @inner-border-color;
+                        padding-left: 6px;
+                        padding-right: 22px;
+                        margin-left: -5px;
+                        -webkit-appearance: none;
+                        -moz-appearance: none;
+                        border-radius: 0;
+                        background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' version='1…0px'><text x='0' y='15' fill='black' font-size='15'>&#x25BE;</text></svg>");
```

Comment:
As far as I know, that's the best solution to style a <select>.
Otherwise you can wrap it in a div and add an :after pseudo element with a unicode arrow content.

Not much cleaner :)